index.html

```diff
@@ -14094,8 +14094,11 @@ <h2>WAI-ARIA Role, State, and Property Quick Reference</h2>
 	<section class="appendix informative section" id="acknowledgements">
 		<h3>Acknowledgments</h3>
 		<p>The following people contributed to the development of this document.</p>
+		<ul id="gh-contributors">
```

Comment:
I'm guessing the change in position was intentional. Either way, I like the change.